src/Workspaces/SharedUtilitiesAndExtensions/Workspace/CSharp/LanguageServices/CSharpTypeInferenceService.TypeInferrer.cs

```diff
@@ -395,6 +287,89 @@ private IEnumerable<TypeInferenceInfo> InferTypeInConstructorInitializer(Constru
                 return InferTypeInArgument(index, methods, argument, parentInvocationExpressionToTypeInfer: null);
             }
 
+            private IEnumerable<TypeInferenceInfo> InferTypesWorkerFromParent(SyntaxNode parent, SyntaxNode node, ExpressionSyntax expression, SyntaxToken? token)
+            {
+                return parent switch
+                {
+                    // Path from SyntaxNode, node is never null
```

Comment:
if you're interested, you can remove `#nullable disable` *in another PR*, and then fix all this up.

Reply:
i will work on this as another PR. 
Thanks for the input.